src/software/ai/hl/stp/tactic/mutable_tactic_visitor.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+// We forward-declare all the tactics because if we include them we induce a
+// circular dependency between the Individual library for each tactic and this
+// visitor. Ex: `CherryPickTactic` includes `TacticVisitor`, but `TacticVisitor`
+// also includes `CherryPickTactic`
+class CherryPickTactic;
+class ShadowFreekickerTactic;
+class GoalieTactic;
+class CreaseDefenderTactic;
+class ShadowEnemyTactic;
+class MoveTactic;
+class ChipTactic;
+class KickoffChipTactic;
+class StopTactic;
+class PatrolTactic;
+class PenaltyKickTactic;
+class PenaltySetupTactic;
+class ReceiverTactic;
+class ShootGoalTactic;
+class PasserTactic;
+class DefenseShadowEnemyTactic;
+class MoveTestTactic;
+class StopTestTactic;
+
+
+/**
+ * This class provides a mutable interface for all Tactic Visitors. The Visitor design
+ * pattern allows us to perform operations on Tactic objects without needing to check
+ * which concrete type it is with an if/else statement, and we don't need to pollute the
+ * Tactic classes with information or functions that are specific to the task we
+ * want to perform. The mutable version allows us to update parameters within
+ * each Tactic.
+ */
```

Comment:
I think we can remove the note here on why we use the visitor design pattern. At this point we've put that sort of thing in the docs. 
(ie. " The Visitor design
 * pattern allows us to perform operations on Tactic objects without needing to check
 * which concrete type it is with an if/else statement, and we don't need to pollute the
 * Tactic classes with information or functions that are specific to the task we
 * want to perform.")

Reply:
changed it to refer to docs

Reply:
changed it to refer to docs